src/PoshArmDeployment/Public/ResourceProviders/Microsoft.Network/ApplicationGateway/Add-ArmApplicationGatewayUrlMapPath.ps1

```diff
@@ -0,0 +1,46 @@
+function Add-ArmApplicationGatewayUrlMapPath {
+    [CmdletBinding(SupportsShouldProcess = $true)]
+    [OutputType("ApplicationGateway")]
+    Param(
+        [PSTypeName("ApplicationGateway")]
+        [Parameter(Mandatory, ValueFromPipeline)]
+        $ApplicationGateway,
+        [ValidatePattern('^(\[.*\]|[a-zA-Z0-9-]*)$')]
+        [string]
+        [Parameter(Mandatory)]
+        $Name,
+        [Parameter(Mandatory)]
+        [string]
+        $BackendAddressPoolName,
+        [Parameter(Mandatory)]
+        [string]
+        $BackendHttpSettingsName,
```

Comment:
Not sure they are mandatory

Reply:
I will try deploying without them

Reply:
It didn't work so I had to add the default behavior to set backendpool and httpsettings